### DISCUSSION TITLE
Can thonny increase the default height of the display area of debug variables?

### DISCUSSION THREAD

**Discussion by wilikeit** [thonny/thonny#3382] [General]
The display area of debug variables is too narrow, and it is not very convenient for user， I need to drag the window every time.  Can thonny increase the default height of the display area of debug variables?
![image](https://github.com/user-attachments/assets/a3b04a30-7460-4061-830c-260a5c37908c)

**Comment by aivarannamaa** [comment]
Did you have more than 2 local variables in this case? If yes, then please create an issue which reminds me to look into this for Thonny 5.0.

**Reply by wilikeit** [reply]
OK, I have already created an issue.
